Adds AUTHORS and NEWS files to make auto-reconf happy.
--- /dev/null
+++ b/AUTHORS
@@ -0,0 +1 @@
+Aaron Brown <aaron@internet2.edu>
--- /dev/null
+++ b/NEWS
@@ -0,0 +1 @@
+Placeholder
--- a/configure.ac
+++ b/configure.ac
@@ -81,7 +81,6 @@
 AC_CHECK_HEADERS([errno.h getopt.h netdb.h stdlib.h tgmath.h sys/param.h sys/socket.h sys/time.h sys/timex.h])
 
 # Checks for typedefs, structures, and compiler characteristics.
-I2_C___ATTRIBUTE__
 AC_C_CONST
 AC_HEADER_TIME
 AC_CHECK_MEMBERS([struct sockaddr.sa_len], , ,
